packages/subgraph/schema.graphql

```diff
@@ -0,0 +1,8 @@
+type PoolMetadataUpdated @entity(immutable: true) {
+  id: Bytes!
+  poolId: Bytes! # bytes32
```

Comment:
can you add proper documentation for every entity and field? example: https://github.com/balancer/gauges-subgraph/blob/master/schema.graphql